packages/create-skill-app/templates/next/src/trpc/routers/_app.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  mergeRouters,
+  router,
+  skillLessonRouter,
+} from '@skillrecordings/skill-lesson'
+import {abilities} from './abilities'
+
+export const appRouter = mergeRouters(
+  router({
+    abilities: abilities,
+  }),
+  skillLessonRouter,
+)
```

Comment:
I figure an ability check will be a pretty standard part of any app we stand up, so that is what is being provided by this base `tRPC` setup.